Source/MoreDotNet.Test/Extensions/Numeric/RomanNumeralExtensions/IsValidRomanNumeralTests.cs

```diff
@@ -0,0 +1,212 @@
+            Assert.True("XLV".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenL_ShouldReturnTrue()
+        {
+            Assert.True("L".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenLXVIII_ShouldReturnTrue()
+        {
+            Assert.True("LXVIII".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenLXXXIII_ShouldReturnTrue()
+        {
+            Assert.True("LXXXIII".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenXCVII_ShouldReturnTrue()
+        {
+            Assert.True("XCVII".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenXCIX_ShouldReturnTrue()
+        {
+            Assert.True("XCIX".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenC_ShouldReturnTrue()
+        {
+            Assert.True("C".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenD_ShouldReturnTrue()
+        {
+            Assert.True("D".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenDI_ShouldReturnTrue()
+        {
+            Assert.True("DI".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenDCXLIX_ShouldReturnTrue()
+        {
+            Assert.True("DCXLIX".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenDCCXCVIII_ShouldReturnTrue()
+        {
+            Assert.True("DCCXCVIII".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenDCCCXCI_ShouldReturnTrue()
+        {
+            Assert.True("DCCCXCI".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenM_ShouldReturnTrue()
+        {
+            Assert.True("M".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenMIV_ShouldReturnTrue()
+        {
+            Assert.True("MIV".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenMVI_ShouldReturnTrue()
+        {
+            Assert.True("MVI".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenMXXIII_ShouldReturnTrue()
+        {
+            Assert.True("MXXIII".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenMMXIV_ShouldReturnTrue()
+        {
+            Assert.True("MMXIV".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenMMMCMXCIX_ShouldReturnTrue()
+        {
+            Assert.True("MMMCMXCIX".IsValidRomanNumeral());
+        }
+
+        [Fact]
+        public void IsValidRomanNumeral_GivenEmptyString_ShouldReturnFalse()
```

Comment:
Again - please merge all false cases into one using the XUnit Data Driven Test approach.